CONTRIBUTING.md

```diff
@@ -216,7 +216,11 @@ suggestions.
   to configure your favorite editor to have `fourmolu` automatically
   applied to your code; but if you don't know how to set that up, don't
   worry!  The [`restyled.io` bot](https://restyled.io/) automatically
-  reformats pull requests as necessary.
+  reformats pull requests as necessary. Here's the command to apply
+  the formatting that *exactly* matches what the bot would do:
+
+      cabal install fourmolu-0.4.0.0
+      find src/ app/ test/ -name "*.hs" | xargs fourmolu --mode=inplace --cabal-default-extensions
```

Comment:
I was not aware you also added it here, @valyagolev. :laughing: Meanwhile, I opened #744.

Technically this is not true - because of our restyled configuration file from times of Fourmolu 0.3 it runs with:
```bash
find src/ app/ test/ -name "*.hs" | xargs fourmolu --mode=inplace '--ghc-opt=-XBangPatterns' \
         '--ghc-opt=-XFlexibleContexts' \
         '--ghc-opt=-XFlexibleInstances'\
         '--ghc-opt=-XGADTSyntax'\
         '--ghc-opt=-XMultiParamTypeClasses'\
         '--ghc-opt=-XNumericUnderscores'\
         '--ghc-opt=-XRankNTypes'\
         '--ghc-opt=-XTupleSections'\
         '--ghc-opt=-XTypeApplications'\
         '--ghc-opt=-XTypeOperators'\
         '--ghc-opt=-XImportQualifiedPost'\
         '--ghc-opt=-XLambdaCase'
```
Which should be the same, but I should probably update it anyway.

While it is nice that you noted it down, it's kind of silly to put everything in this one PR, please feel welcome to open other small PRs. :wink:

Reply:
oh that explains why i had to manually undo some of its changes then :)

I just copied what I was suggested. I will remove it, thanks for your PR

Reply:
Yes, you will have to revert the change to this file, because I merged first, sorry about that. :sweat_smile: 

You can do that automatically but in this case, it's quite easy either way.